views/js/review/plugins/content/item-answer/item-answer.js

```diff
@@ -54,6 +56,8 @@ define([
         skippedText: __('No response'),
         scoreText: __('Your Score:'),
         status: 'informational',
+        showScore: true,
+        showCorrect: true,
```

Comment:
here default value is true for showScore and showCorrect

Reply:
Indeed, maybe should I turn those values to false. The reason why I didn't is I first wanted to keep compatibility with existing feature, and then didn't want to rework too much the unit tests, since they rely on the former behavior.